codespell_lib/data/dictionary.txt

```diff
@@ -13945,6 +13974,7 @@ unbeleivable->unbelievable
 unbeliefable->unbelievable
 unbelivable->unbelievable
 unce->once
+uncehcked->unchecked
```

Comment:
I would also add above it `uncehck->uncheck`

Reply:
ok.